### PR TITLE
MEEd-114: Edit program

### PR DESCRIPTION
**Pull request by GouadriaHanen**
prior to this change, the manual program could not be edited
after this change, the program owner is able to edit his programs

### CODE REVIEW

File: portlets/src/main/webapp/vue-app/engagement-center/components/common/ImageSelector.vue [hunk 1]
```diff
@@ -75,8 +69,8 @@ export default {
   },
 
   computed: {
-    coverUrl() {
-      return this.coverData || this.cover;
+    cover() {
```
Comment (AzmiTouil) [inline portlets/src/main/webapp/vue-app/engagement-center/components/common/ImageSelector.vue:76]:
```suggestion
    coverUrl() {
```

File: services/src/main/java/org/exoplatform/addons/gamification/service/mapper/DomainMapper.java [hunk 2]
```diff
@@ -76,9 +76,8 @@ public static DomainDTO domainEntityToDomainDTO(DomainEntity domainEntity, Domai
     if (domainEntity == null) {
       return null;
     }
-    long lastUpdateTime = domainEntity.getCoverFileId() == 0
-        && domainEntity.getLastModifiedDate() == null ? 0 : domainEntity.getLastModifiedDate().getTime();
-    String coverUrl = Utils.buildAttachmentUrl(String.valueOf(domainEntity.getId()), lastUpdateTime, Utils.TYPE);
+    long lastUpdateTime = domainEntity.getLastModifiedDate().getTime();
```
Comment (AzmiTouil) [inline services/src/main/java/org/exoplatform/addons/gamification/service/mapper/DomainMapper.java]:
```suggestion
    long lastUpdateTime = domainEntity.getLastModifiedDate() == null ? 0 : domainEntity.getLastModifiedDate().getTime();
```

**Review by AzmiTouil — CHANGES_REQUESTED** [review]

File: portlets/src/main/webapp/vue-app/engagement-center/components/common/ImageSelector.vue (hunk 1 above)
Comment (GouadriaHanen) [reply portlets/src/main/webapp/vue-app/engagement-center/components/common/ImageSelector.vue:76]: changed in all the component to cover!

File: services/src/main/java/org/exoplatform/addons/gamification/service/mapper/DomainMapper.java (hunk 2 above)
Comment (GouadriaHanen) [reply services/src/main/java/org/exoplatform/addons/gamification/service/mapper/DomainMapper.java]: done!

**Review by AzmiTouil — APPROVED** [review]

**Pull request merged**